index-fr.html

```diff
@@ -52,52 +52,6 @@ <h3>Nous collaborons afin de...</h3>
 	</section>
 </div>
 
-<section class="section section--featured-video">
```

Comment:
Nice, that video was getting kind of stale

Reply:
Coming up on a year 🎆